client/components/tinymce/plugins/media/restrict-size/index.js

```diff
@@ -58,6 +70,15 @@ export default function( editor ) {
 		}
 
 		event.content = setImages( event.content );
+
+		/**
+		 * Also add the raw resized image URL so we can properly preload it in the editor
+		 */
+		const resizedUrl = getResizedImgUrlFromImgString( event.content );
```

Comment:
I'm thinking we might not want to be calling this on the `BeforeSetContent` event, which is the full string content of the post and may contain mixture of text and zero, one, or more images. Seems wasteful for something we only intend to be used in response to `BeforeSetWpcomMedia`. Maybe we should create a separate event handler for this bit.